en/docs/chapter_graph/graph.md

```diff
@@ -55,8 +55,8 @@ As shown in the figure below, let the adjacency matrix be $M$, and the list of v
 Adjacency matrices have the following characteristics.
 
 - A vertex cannot be connected to itself, so the elements on the main diagonal of the adjacency matrix are meaningless.
-- For undirected graphs, edges in both directions are equivalent, thus the adjacency matrix is symmetric about the main diagonal.
-- By replacing the elements of the adjacency matrix from $1$ and $0$ to weights, it can represent weighted graphs.
+- For undirected graphs, edges in both directions are equivalent, thus the adjacency matrix is symmetric with regard to the main diagonal.
+- By replacing the elements of the adjacency matrix from $1$ and $0$ to weight values, it can represent weighted graphs.
```

Comment:
1/ "weights" is commonly used to describe the number value of graph, so I don't think it's necessary to replace it with "weight values".
2/ subject-predicate agreement principle: the subject of the sentence should be "we" instead of "it".

Reply:
Taken